data/json/npcs/hints.json

```diff
@@ -70,22 +79,23 @@
     "Be careful eating wild mushrooms.  Some are poisonous, and others can make you hallucinate.",
     "Try to go around swamps, if you can.  Some of them have sinkholes that can pull you right underground.",
     "I heard about this group that raided a bee hive a while ago.  Everyone got massacred but one, and he came home with this weird, magic honey stuff.",
-    "If you need to, you can swim across a river to escape; most monsters can't swim.  Just make sure you drop as much stuff as possible first, and maybe strip naked, or you'll sink like a rock.",
-    "Houses can be a surprisingly good resource for all kinds of stuff; clothing, medication, food, books, and more.",
+    "If you need to, you can swim across a river to escape; some monsters can't swim.  Just make sure you drop as much stuff as possible first, and maybe strip naked, or you'll sink like a rock.",
+    "Houses can be a surprisingly good resource for all kinds of stuff; clothing, medication, food, books, and more.  People kept all the odd things around, especially in basements.",
     "While there's not much to loot, gas stations are a good source for gasoline, to power tools or to make Molotov cocktails.",
-    "It's not like in the movies--shooting a gas pump won't make it explode.  But it WILL make it leak all over the place, which is a definite fire hazard.",
+    "It's not like in the movies - shooting a gas pump won't make it explode.  But it WILL make it leak all over the place, which is a definite fire hazard.",
     "I know grocery stores and stuff are full of fruit, but be careful, it's probably rotten.",
     "Next time you visit a grocery store, load up on canned goods.  They never go bad!",
     "I've found more good weapons in hardware stores than anywhere else.  Except gun stores, of course.",
     "Liquor stores are a great place to pick up non-alcoholic beverages, too.  Not that I mind alcohol!",
-    "Most gun stores follow pretty similar layouts.  The restricted stuff--SMGs, assault rifles, and most importantly ammo--are always behind the counter.",
+    "Most gun stores follow pretty similar layouts.  The restricted stuff - SMGs, assault rifles, and most importantly ammo - are always behind the counter.",
     "I've spent many nights in the changing rooms at the back of clothing stores.  Walls on four sides, far from the store's entrance, a corridor for easy defense... it's perfect!",
     "If you're getting chased by a hoard of zombies, try ducking into the subways and traveling a block or two, then coming back up.",
     "Have you seen those weird science labs out in the middle of nowhere?  I think you need some kind of special ID card to get in.",
+    "I have a dream of raiding a military bunker, but I well know that good protection plus zombie soldiers are a bad mix.  The thought of zombie grenadiers alone is giving me the shivers.",
```

Comment:
We don't have zombie grenadiers anymore.

Reply:
Wait what? I thought they were just pushed to the deep corners of the military bunkers. Are they totally removed?

Reply:
Quote from #28664:

>Zombie grenadiers have been replaced by the Northrop Dispatch, a new robot.

Reply:
I was aware of that, but I thought it wasn't total wipe. Oh well. "Flow my tears the policeman said". They will be missed. I'll rework the snippet to mention something about mechanical police brutality.

Reply:
Done.